docs/transactions/iti41.md

```diff
@@ -1,35 +1,229 @@
 # ITI-41
 
-DocumentEntry metadata are described in [documents](documents.md). Additional constraints are described here.
+The [ITI-41 (Provide and Register Document Set-b)](https://profiles.ihe.net/ITI/TF/Volume2/ITI-41.html) transaction allows a [content sender](https://profiles.ihe.net/ITI/TF/Volume2/ITI-41.html#3.41.2) to send documents to the eMedication service ([content receiver](https://profiles.ihe.net/ITI/TF/Volume2/ITI-41.html#3.41.2)). 
+
+Generic [DocumentEntry metadata](https://profiles.ihe.net/ITI/TF/Volume3/ch-4.2.html#4.2.1.1) are described in the [documents section](documents.md). This page references additional constraints specific to ITI-41. See also IHE's documentation for [metadata in Document Sharing profiles](https://profiles.ihe.net/ITI/TF/Volume3/index.html).
+
+## Publishing documents
+### Document types
+Two types of documents might be published to the service :
+- [CH-EMED-EPR](https://build.fhir.org/ig/CARA-ch/ch-emed-epr/) documents. These documents contain the eMedication data. See also the [corresponding page](../emed/index.md) in this guide.
+- [APPC (Advanced Patient Privacy Consent)](https://www.ihe.net/uploadedFiles/Documents/ITI/IHE_ITI_Suppl_APPC.pdf) documents. These documents are used to let patients define who can see their data. See also the [corresponding page](../appc/index.md) in this guide.
```

Comment:
not only see, since authorized HCPs will be able to submit documents for this patient, perhaps something like _define access rights to their data_ or any equivalent/better formula would be better suited